### PR TITLE
updating node and renaming mots referenes to next-template for consis…

### PR DESCRIPTION
**Pull request by P0NDER0SA**
updating node and renaming mots referenes to next-template for consistency

### CODE REVIEW

File: helmfile/helmfile.yaml [hunk 1]
```diff
@@ -15,12 +15,12 @@ environments:
 releases:
 # Single Tier Website
   - name: {{ requiredEnv "APP_NAME" }}-{{ .Environment.Name }}
-    chart: ../helm/Charts/single-tier-appliation
+    chart: ../helm/Charts/next-template
     values:
-      - overrides/single-tier-application/single-tier-application.yaml.gotmpl
+      - overrides/next-template/next-template.yaml.gotmpl
```
Comment (s-laugh) [inline helmfile/helmfile.yaml:20]: The file looks to be renamed to `.../next-template.gotmpl` without the `.yaml`

**Review by shewood — APPROVED** [review]
Looks decent.

File: helmfile/helmfile.yaml (hunk 1 above)
Comment (P0NDER0SA) [reply helmfile/helmfile.yaml:20]: that's weird :/  nice catch though, jeezus

Comment (s-laugh) [reply helmfile/helmfile.yaml:20]: I think I confused you.... looks like it should be `.../next-template.yaml.gotmpl`; sorry

**Pull request merged**